trusted_os/console.go

```diff
@@ -40,6 +40,8 @@ import (
 const debug = false
 
 func init() {
+	// disable ARM debug operations
+	imx6ul.Debug(false)
```

Comment:
This is unnecessary on secure booted units and irrelevant on open ones, nonetheless it's good practice to invoke it as an abundance of caution given its null cost.